client/js/views/GameView.vue

```diff
@@ -375,7 +344,11 @@ export default {
     };
   },
   computed: {
+    ...mapState({
+      waitingForOpponentToPlayFromDeck: ({ game }) => game.waitingForOpponentToPlayFromDeck,
+    }),
```

Comment:
This would be the same as above if I'm right.
```suggestion
    ...mapState([
      'waitingForOpponentToPlayFromDeck',
    ]),
```